packages/react-devtools-scheduling-profiler/src/hooks.js

```diff
@@ -57,3 +60,10 @@ export function useBrowserTheme(): void {
     }
   }, [theme]);
 }
+
+export function useDisplayDensity(): void {
+  useLayoutEffect(() => {
+    const documentElements = [((document.documentElement: any): HTMLElement)];
+    updateDisplayDensity('comfortable', documentElements);
+  }, []);
+}
```

Comment:
Fixes more missing CSS variables 😆 I'm using the comfortable density as I think it works better for a standalone app, but let me know if I should change this to compact.

|`master`|With this fix|
|-|-|
|![image](https://user-images.githubusercontent.com/12784593/92245727-58a89f00-eef7-11ea-86dd-6df0341cdd5e.png)|![image](https://user-images.githubusercontent.com/12784593/92245766-64946100-eef7-11ea-88f8-3518a240468e.png)|

Reply:
👍